app/Http/Controllers/MemberController.php

```diff
@@ -12,63 +12,70 @@
 
 class MemberController extends Controller
 {
-    public function listMember(Request $request)
+    public function index(Request $request)
     {
-        $listMember= Member::all()->toArray();
+        $listMember = Member::all() ->toArray();
         return $listMember ;
     }
 
-    public function deleteMember($id)
+    public function create()
```

Comment:
If you doesn't using this function. Pls remove it.

Reply:
i have remove it